pkg/sql/catalog/descs/collection_test.go

```diff
@@ -152,23 +152,11 @@ func TestTxnClearsCollectionOnRetry(t *testing.T) {
 	ctx := context.Background()
 
 	const txnName = "descriptor update"
-	haveInjectedRetry := false
 	var serverArgs base.TestServerArgs
 	params := base.TestClusterArgs{ServerArgs: serverArgs}
+	filterFunc, _ := testutils.TestingRequestFilterRetryTxnWithPrefix(t, txnName, 1)
```

Comment:
don't you always want to verifyFunc() at the end of the test? otherwise maybe you didn't test what you wanted? (I know it's not related to your pr..)